modules/code-editor/src/views/full/FileNavigator.tsx

```diff
@@ -39,10 +39,17 @@ class FileNavigator extends React.Component<Props, State> {
     observe(this.props.filters, 'filename', this.refreshNodes, true)
   }
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps: Props) {
     if (this.props.files && prevProps.files !== this.props.files) {
       this.refreshNodes()
     }
+    if (this.props.selectedNode !== prevProps.selectedNode) {
+      const { nodes } = this.state
+      let { selectedNode } = this.props
+      selectedNode = selectedNode.replace(`${this.props.id}/`, '')
```

Comment:
The temp variable isn't necessary: ``const selectedNode =  this.props.selectedNode.replace(`${this.props.id}/`, '')``